### PR TITLE
deposit: deposit migration fix

### PR DESCRIPTION
**Pull request by krzysztof**
* Fixes the deposit migration using marshmallow schemas.

Signed-off-by: Krzysztof Nowak <k.nowak@cern.ch>

### CODE REVIEW

File: requirements-devel.txt [hunk 1]
```diff
@@ -26,4 +26,6 @@
 -e git+https://github.com/inveniosoftware/invenio-deposit.git#egg=invenio-deposit
 -e git+git://github.com/inveniosoftware/invenio-files-rest.git#egg=invenio-files-rest
 -e git+git://github.com/inveniosoftware/invenio-indexer.git#egg=invenio-indexer
-
+-e git+https://github.com/inveniosoftware/invenio-sipstore.git#egg=invenio-sipstore
+-e git+https://github.com/zenodo/zenodo-accessrequests.git#egg=zenodo-accessrequests
+-e git+git://github.com/zenodo/zenodo.git#egg=zenodo
```
Comment (lnielsen) [inline requirements-devel.txt]: Aren't we going to have the same circular dependencies as before?

Comment (krzysztof) [inline requirements-devel.txt]: I don't think so as Zenodo has migrationkit only as a development dependency only.

**Comment by krzysztof** [comment]
Tests on this one will pass as soon as this is merged: https://github.com/zenodo/zenodo/pull/649 (Same flask 0.11 issues as on Zenodo)

**Comment by krzysztof** [comment]
As far as I can tell :shipit:

**Comment by lnielsen** [comment]
:shipit:

**Pull request merged**